monkey/monkey_island/cc/next_ui/src/_components/app-nav/app-menu/AppMenu.tsx

```diff
@@ -1,44 +1,49 @@
 import { PATHS } from '@/constants/paths.constants';
-import Typography from '@mui/material/Typography';
-import Box from '@mui/material/Box';
 import React from 'react';
-import { useRouter } from 'next/navigation';
-import classes from './appMenu.module.scss';
+import { usePathname, useRouter } from 'next/navigation';
 import HomeIcon from '@mui/icons-material/Home';
 import HubIcon from '@mui/icons-material/Hub';
-import EventNoteIcon from '@mui/icons-material/EventNote';
+import Tab from '@mui/material/Tab';
+import Tabs from '@mui/material/Tabs';
 
 const MenuLinks = [
-    { path: PATHS.HOME, label: 'Home', icon: <HomeIcon /> },
-    { path: PATHS.MAP, label: 'Map', icon: <HubIcon /> },
-    { path: PATHS.EVENTS, label: 'Events', icon: <EventNoteIcon /> }
+    { path: PATHS.CONFIGURE, label: 'Configure', icon: <HomeIcon /> },
+    { path: PATHS.RUN, label: 'Run', icon: <HomeIcon /> },
+    { path: PATHS.NETWORK_MAP, label: 'Network Map', icon: <HubIcon /> },
+    { path: PATHS.REPORT, label: 'Report', icon: <HomeIcon /> }
 ];
 
 export interface MenuProps {
     onClose?: () => void;
+    orientation?: 'vertical' | 'horizontal';
 }
 
-const AppMenu = ({ onClose }: MenuProps) => {
+const AppMenu = ({ orientation, onClose }: MenuProps) => {
     const router = useRouter();
+    const path = usePathname();
 
-    const handleRouteClick = (path: string) => {
-        router.push(path);
+    const handleRouteClick = (event: React.SyntheticEvent, value: any) => {
+        router.push(value);
         onClose && onClose();
     };
 
     return (
-        <Box className={classes['app-menu']}>
-            <Box className="menu-links">
-                {MenuLinks.map((link) => (
-                    <div
-                        key={link.label}
-                        className={'app-route-link'}
-                        onClick={() => handleRouteClick(link.path)}>
-                        <Typography>{link.label}</Typography>
-                    </div>
-                ))}
-            </Box>
-        </Box>
+        <Tabs
+            orientation={orientation}
+            value={path}
```

Comment:
If the path is ROOT ("/") then this component starts throwing errors, because there's no tab defined for this value. We need to circumvent this somehow